front/lib/api/assistant/conversation.ts

```diff
@@ -1323,7 +1323,6 @@ export async function retryAgentMessage(
   const agentMessageArray = conversation.content.find((messages) => {
     return messages.some((m) => m.sId === message.sId && isAgentMessageType(m));
   }) as AgentMessageType[];
-  agentMessageArray.push(agentMessage);
```

Comment:
agentMessage is already added a few lines below, when the conversation is stiched

https://github.com/dust-tt/dust/blob/5a1ce9343b7441149a2e3c104eb1629cebbe1778/front/lib/api/assistant/conversation.ts#L1327-L1331